sass/site/primary/_posts-and-pages.scss

```diff
@@ -20,10 +20,43 @@
 	display: inline;
 }
 
+.page,
+.single {
+
+	.entry-header {
+		background: $blue;
+		color: $white;
+		margin-bottom: 95px;
```

Comment:
Is this meant to be the height of the wave, or is this an arbitrary margin?

Reply:
Yep, you're right. Updated to a variable.